### PR TITLE
handle the empty MMS case ( any errors while retrieving ) 

### PR DESCRIPTION
**Pull request by lduboeuf**
Pre-requisite: https://github.com/ubports/nuntium/pull/3

in conversation view:

screenshot20201101_224426210

in thread list:
screenshot20201102_104752724

### CODE REVIEW

**Comment by lduboeuf** [comment]
screenshot20201029_122841116

**Comment by lduboeuf** [comment]
screenshot20201029_124311972

**Comment by Flohack74** [comment]
Are there any linked issues that will be closed with that PR?

**Comment by lduboeuf** [comment]
@Flohack74 that one is related but not really fixed as the mms still lost: https://github.com/ubports/ubuntu-touch/issues/1408 and that one: https://github.com/ubports/ubuntu-touch/issues/239

**Comment by lduboeuf** [comment]
and please don't forget https://github.com/ubports/telephony-service/pull/13
for the notification part

**Comment by Flohack74** [comment]
Please ping me on Telegram, for the order and which issues I need to mark for OTA-15. I need to do the admin stuff on this one too ^^

File: src/qml/ThreadDelegate.qml [hunk 1]
```diff
@@ -24,13 +24,18 @@ import Ubuntu.Telephony 0.1
...
+
 import "dateUtils.js" as DateUtils
 
 ListItemWithActions {
     id: delegate
 
     property QtObject chatEntry: null
     property bool compactView: false
+    property bool isEmptyMMS: eventTextMessageType === HistoryEventModel.MessageTypeMultiPart && displayedEventTextAttachments.length === 0 && eventTextMessage.length === 0
+    property string defaultMMSErrorMessage: i18n.tr("Oops, there has been an error with the MMS system and this message could not be retrieved.
```
Comment (Flohack74) [inline src/qml/ThreadDelegate.qml]: Why is this needed here, isnt that the same as https://github.com/ubports/telephony-service/pull/13/files#diff-e65b0117bee632c68e29f9596763e0867a876da2ed407c71ab9c6752e54c7f77R504 ?

Comment (lduboeuf) [reply src/qml/ThreadDelegate.qml]: yes the same, but translation can't be shared between those two afaik ?

File: po/messaging-app.pot [hunk 2]
```diff
@@ -534,10 +535,18 @@ msgstr ""
...
 msgstr ""
 
+#: ../src/qml/MessageAlertBubble.qml:68 ../src/qml/ThreadDelegate.qml:37
+msgid ""
+"Oops, there has been an error with the MMS system and this message could not "
+"be retrieved."
+msgid_plural ")"
+msgstr[0] ""
+msgstr[1] ""
+
```
Comment (lduboeuf) [inline po/messaging-app.pot:548]: There is an issue here with translation

File: src/qml/ThreadDelegate.qml (hunk 1 above)
Comment (Flohack74) [reply src/qml/ThreadDelegate.qml]: No, translation cannot be shared. But if you use the exact same string the translation server can update it in both :)

Comment (lduboeuf) [reply src/qml/ThreadDelegate.qml]: Oh, so tell me how to do it ?, translation will be exactly the same

Comment (lduboeuf) [reply src/qml/ThreadDelegate.qml]: @Flohack74 can we go ahead for that PR ?

**Review by Flohack74 — APPROVED** [review]

**Comment by UniversalSuperBox** [comment]
I've received quite a number of these errors in the past few days, so I agree that this notification functionality is working.
